src/DataTableAbstract.php

```diff
@@ -249,12 +249,21 @@ public function escapeColumns($columns = '*')
     /**
      * Set columns that should not be escaped.
      *
+     * Optionally merge the defaults from config
```

Comment:
Missing period `.` and please remove the new line before this comment.

Reply:
No probs, updated.